### ISSUE TITLE
Is this feature from another project?

### ISSUE THREAD

**Issue by AlphaHot** [ygongdev/jigsaw-canvas#2]
https://github.com/ygongdev/jigsaw-canvas/blob/f4525bd6b3f838ee405a68edbf50f15918bf4e5c/index.js#L263-L268

**Comment by ygongdev** [comment]
No, it was used to try to get the calculated peak height of the jigsaw pieces. Each piece actually should be of different width and dimensions, but to simplify, every piece has the same width and height, so there is white space included. The peak calculation will give the correct width and height of a piece if used but need to figure out how to use it properly.
